infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/log/PodLogHandlerToEventPublisher.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.workspace.infrastructure.kubernetes.namespace.log;
+
+import java.time.ZonedDateTime;
+import org.eclipse.che.api.core.model.workspace.runtime.RuntimeIdentity;
+import org.eclipse.che.workspace.infrastructure.kubernetes.util.RuntimeEventsPublisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** This class is responsible for reading the logs. It is aware of machines it should follow. */
+public class PodLogHandlerToEventPublisher implements PodLogHandler {
```

Comment:
I think `PodLogToEventPublisher` is a better name for this class, the current name makes it sound like it converts a `PodLogHandler` into an `EventPublisher`, rather than converting pod logs to events.

Reply:
yes